python/podio_class_generator.py

```diff
@@ -299,21 +310,26 @@ def _preprocess_for_class(self, datatype):
     datatype['includes'] = self._sort_includes(includes)
     datatype['includes_cc'] = self._sort_includes(includes_cc)
     datatype['forward_declarations'] = fwd_declarations
+    datatype['includes_jl'] = set()
+    datatype['includes_jl'].update(self._sort_includes(includes_jl))
 
   def _preprocess_for_collection(self, datatype):
     """Do the necessary preprocessing for the collection"""
-    includes_cc, includes = set(), set()
+    includes_cc, includes, includes_jl = set(), set(), set()
 
     for relation in datatype['OneToManyRelations'] + datatype['OneToOneRelations']:
       if datatype['class'].bare_type != relation.bare_type:
         includes_cc.add(self._build_include(relation.bare_type + 'Collection'))
+        includes_jl.add(self._build_include(relation.bare_type + 'Collection', julia=True))
```

Comment:
Is this necessary? Respectively can you use the generated files with this? I am asking because as far as I can see there is no `XYZCollection.jl` that we generate here(?)